crypto/sha.go

```diff
@@ -5,18 +5,18 @@
 package crypto
 
 import (
-	"github.com/seeleteam/go-seele/common"
 	"github.com/seeleteam/go-seele/crypto/sha3"
 )
 
 // Keccak256Hash calculates and returns the Keccak256 hash of the input data,
 // converting it to an internal Hash data structure.
-func Keccak256Hash(data ...[]byte) (h common.Hash) {
+func Keccak256Hash(data ...[]byte) []byte {
 	d := sha3.NewKeccak256()
 	for _, b := range data {
 		d.Write(b)
 	}
 
+	h := make([]byte, 32)
```

Comment:
nice to define this magic number to a constant.

Reply:
good suggestion